cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -957,6 +959,10 @@ export function getJobAsync(tid: number, jid: number, initialFrame: number, init
                     maxZ,
                 },
             });
+            if (job.task.dimension === '3d') {
```

Comment:
`job.task.dimension === DimensionType.DIM_3D`
Preferred to use enums instead of plain strings